server/notebooks/views.py

```diff
@@ -186,3 +187,36 @@ def tryit_view(request):
             "eval_frame_origin": EVAL_FRAME_ORIGIN,
         },
     )
+
+'''
+    Let user use his own template file and data
+    template and data should be enclosed in a POST request.
+'''
+@csrf_exempt
+def customize_view(request):
+
+    if request.method == "POST":
+        
+        postedTemplate = request.FILES['template']
+        postedData = request.FILES['data']
+    
+        template = postedTemplate.read().decode()
+        data = postedData.read().decode()
+
+        iomd = template + data
+
+        return render(
```

Comment:
This doesn't look right to me. What I'd expect is a response that looks like this:

```html
<script id="iomd" type="text/iomd">%% md
example md content
</script>
<script id="file0" type="application/json">
{
 "a": 1,
 ...
}
</script>
```

Putting the response into the script tag will let us easily extract and process the response into a file from javascript.